### DISCUSSION TITLE
Mobile app Fork?

### DISCUSSION THREAD

**Discussion by a2937** [freeCodeCamp/Developer_Quiz_Site#626] [Ideas]
I wanna see if it's possible to use React Native so it can be accessible on individual's phones along with the web. Mobile apps are relatively popular in my opinion and having this developer quiz site as a companion application the game could be pretty cool?

**Comment by jdwilkin4** [comment]
At this time we are not looking to add a mobile app for the Developer Quiz Site. 
If anything changes, then I make an update in the discussions here
